make/Main.gmk

```diff
@@ -753,6 +753,17 @@ $(eval $(call SetupTarget, test-image-lib, \
     DEPS := build-test-lib, \
 ))
 
+$(eval $(call SetupTarget, build-test-setup-aot, \
+    MAKEFILE := test/BuildJtregSetupAOT, \
+    DEPS := interim-langtools exploded-image build-test-lib, \
```

Comment:
I don't think `build-test-lib` is needed.

Reply:
Removed.